index.html

```diff
@@ -2371,99 +2469,30 @@ <h3 id=new-session><dfn data-lt="new sessions|creating a new session">New Sessio
-   <dd><var>session id</var>
+   <dd><var>session</var>'s <a>session ID</a>.
 
    <dt>"<code>capabilities</code>"
    <dd><var>capabilities</var>
   </dl>
 
- <li><p>Initialize the following from <var>capabilities</var>:
-
-  <ol>
-   <li><p>Let <var>strategy</var> be the result of getting property
-    "<code>pageLoadStrategy</code>" from <var>capabilities</var>.
-
-   <li><p>If <var>strategy</var> is a string, set the <a>current session</a>’s
-    <a>page loading strategy</a> to <var>strategy</var>.  Otherwise,
-    set the <a>page loading strategy</a> to <i>normal</i> and <a>set a
-    property</a> of <var>capabilities</var> with name
-    "<code>pageLoadStrategy</code>" and value "<code>normal</code>".
-
-   <li><p>Let <var>strictFileInteractability</var> be the result of getting property
-    "<code>strictFileInteractability</code>" from <var>capabilities</var>.
-
-   <li><p>If <var>strictFileInteractability</var> is a boolean,
-    set the <a>current session</a>’s <a>strict file interactability</a> to
-    <var>strictFileInteractability</var>. Otherwise set the
-    <a>current session</a>’s <a>strict file interactability</a> to false.
-
-   <li><p>Let <var>proxy</var> be the result of getting property
-    "<code>proxy</code>" from <var>capabilities</var> and run the
-    substeps of the first matching statement:
-
-       <dl class=switch>
-         <dt><var>proxy</var> is a <a>proxy configuration</a> object</dt>
-         <dd><p>Take implementation-defined steps to set the user agent proxy
-             using the extracted <var>proxy</var> configuration.  If the
-             defined proxy cannot be configured return <a>error</a> with
-             <a>error code</a> <a>session not created</a>.</dd>
-
-         <dt>Otherwise</dt>
-         <dd><a>Set a property</a> of <var>capabilities</var> with name
-           "<code>proxy</code>" and a value that is a new JSON <a>Object</a>.</dd>
-       </dl>
-   </li>
-
-   <li><p>If <var>capabilities</var> has a property with the key "<code>timeouts</code>":
-
-    <ol>
-     <li><p>Let <var>timeouts</var> be the result of <a>trying</a>
-      to <a>JSON deserialize</a> as a <a>timeouts configuration</a>
-      the value of the "<code>timeouts</code>" property.
-
-     <li><p>Make the <a>session timeouts</a> the new <var>timeouts</var>.
-    </ol>
-
-   <li><p>Set a property on <var>capabilities</var> with name
-    "<code>timeouts</code>" and value that of the
-    <a>JSON deserialization</a> of the <a>session timeouts</a>.
-
-   <li><p>Apply changes to the user agent for any implementation-defined capabilities
-    selected during the <a>capabilities processing</a> step.
-  </ol>
-
- <li><p>Set the <a>webdriver-active flag</a> to true.</p>
-
- <li><p><a>Set the current top-level browsing context</a>
-   for <var>session</var> with the <a>top-level browsing context</a>
+ <li><p>Set the <a>current top-level browsing context</a>
+   of <var>session</var> to the <a>top-level browsing context</a>
```

Comment:
should we add: `of the currently selected tab`? Otherwise it would be unclear in case the browser starts up with multiple tabs already open.